amundsen_application/static/js/components/DashboardPage/constants.ts

```diff
@@ -6,5 +6,7 @@ export const OWNER_HEADER_TEXT = 'Owners';
 export const ADD_DESC_TEXT = 'Add Description in';
 export const EDIT_DESC_TEXT = 'Click to edit description in';
 
+export const MISSED_RUN_TEXT = 'Missed';
+export const HIT_RUN_TEXT = 'Hit';
+
 export const LAST_RUN_SUCCEEDED = 'succeeded';
```

Comment:
I think this needs to be capitalized now? Before I think the `Flag` component converted it. We also aren't changing what text this component shows, it should still use these constants...unless Design has changed the Dashboard UI text to `Hit` and `Missed`?

Reply:
Design was changed to Hit and Missed according to the mocks.

Reply:
Can you paste in slack? I see the mock, but I'm only seeing Hit and Missed on the table page.